src/Test/L0/Listener/CommandSettingsL0.cs

```diff
@@ -28,7 +29,7 @@ public void GetsArg()
             using (TestHostContext hc = CreateTestContext())
             {
                 // Arrange.
-                var command = new CommandSettings(hc, args: new string[] { "--agent", "some agent" });
+                var command = new CommandSettings(hc, args: new string[] { "configure", "--agent", "some agent" });
```

Comment:
I notice that the "default verb" is run now. Did that change? Was it "configure" before. Just trying to understand why the tests changed here.

Reply:
Previously any option could be on any verb, even if it was not used by that verb.  The default was "run" but since options were never validated the test didn't have to specify "configure".  Let me know if that makes sense :-).